behave_test_automation/pah_behave/features/steps/steps.py

```diff
@@ -0,0 +1,8 @@
+from behave import given, then, when
```

Comment:
`given` and `then` are not used here

Reply:
This file was not a part of the refactor scope. Will be fixed (and probably removed) in next PR. Only login-testcase steps were refactored.